cylc/flow/parsec/fileparse.py

```diff
@@ -337,7 +337,7 @@ def read_and_proc(fpath, template_vars=None, viewcfg=None, asedit=False):
     # process with EmPy
     if do_empy:
         if (
-            extra_vars['templating_detected'] == 'empy' and
+            extra_vars['templating_detected'] == 'empy:suite.rc' and
```

Comment:
```suggestion
            extra_vars['templating_detected'] == 'empy:flow.cylc' and
```
(should this be changed at both ends?)

Reply:
No. The idea is that the old syntax is maintained for legacy purposes 🐬 , but that the Cylc8 way of doing things is to use 'template variables'. Oliver and I had a very brief discussion of this issue at https://github.com/cylc/cylc-rose/issues/25.

Reply:
Ah, got it, thanks.